src/libNode/FinalBlockProcessing.cpp

```diff
@@ -362,6 +362,8 @@ void Node::ScheduleMicroBlockConsensus() {
 
   auto main_func = [this]() mutable -> void { RunConsensusOnMicroBlock(); };
 
+  SetState(MICROBLOCK_CONSENSUS_PREP);
```

Comment:
This is already set inside `RunConsensusOnMicroBlock` and I sort of remember me and @sandipbhoir noticing this duplicate before and removing it.